tools/ci_build/github/linux/docker/scripts/install_deps.sh

```diff
@@ -28,7 +28,7 @@ fi
 if [ "$INSTALLED_PYTHON_VERSION" = "3.4" ];then
   echo "Python 3.5 and above is needed for running onnx tests!" 1>&2
 else
-  source /tmp/scripts/install_onnx.sh $INSTALLED_PYTHON_VERSION
+  source install_onnx.sh $INSTALLED_PYTHON_VERSION
```

Comment:
I believe this has the same problem as `./install_onnx.sh`

Reply:
Getting abs path ...